plugins/palo_alto_cortex_xdr/plugin.spec.yaml

```diff
@@ -283,10 +1105,34 @@ actions:
 triggers:
   get_incidents:
     title: Get Incidents
-    description: Get Incidents
+    description: Get incidents
+    input:
+      interval:
```

Comment:
Use the `frequency` style here https://docs.rapid7.com/insightconnect/style-guide/#triggers